tests/ui/closures/multiple-fn-bounds.stderr

```diff
@@ -20,6 +18,10 @@ note: required by a bound in `foo`
    |
 LL | fn foo<F: Fn(&char) -> bool + Fn(char) -> bool>(f: F) {
    |                               ^^^^^^^^^^^^^^^^ required by this bound in `foo`
+help: do not borrow the argument
+   |
+LL |     foo(move |char| v);
+   |               ~~~~
```

Comment:
I'm not *adding* this incorrect suggestion, just changing its presentation, but we need to fix this in a follow up.